app/config.rb

```diff
@@ -18,25 +18,31 @@
 AppConfig[:indexer_log] = "/archivesspace/logs/archivesspace.out"
 AppConfig[:indexer_log_level] = "info"
 
-AppConfig[:authentication_sources] = [
-  {
-    model: 'ASOauth',
-    label: 'U-M WebLogin',
-    provider: :openid_connect,
-    config: {
-      issuer: ENV["OIDC_ISSUER"],
-      discovery: true,
-      client_auth_method: 'jwks',
-      scope: [:openid, :email, :profile],
-      client_options: {
-        identifier: ENV["OIDC_CLIENT_ID"],
-        secret: ENV["OIDC_CLIENT_SECRET"],
-        redirect_uri: "#{ENV["PUBLIC_URL"]}/auth/openid_connect/callback"
-      }
-    }
-  }
-]
-
-AppConfig[:plugins] << "aspace-oauth"
 AppConfig[:allow_user_registration] = false
```

Comment:
I was wondering about whether to include this in the no OIDC path or in the `if` block. I guess we can just test using the `admin` user.